frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -131,9 +140,31 @@ Provides asynchronous access to specific modules exposed by the plugin.
 
 `utils_bar_ts-chunk.js`: webpack chunk corresponding to the exposed `barUtils` module.
 
+## Plugin development
+
+Run Bridge locally and instruct it to proxy e.g. `/api/plugins/example-plugin` requests directly to your
+local plugin asset server (web server hosting the plugin's generated assets):
+
+```sh
+./bin/bridge -plugins example-plugin=http://localhost:9001/
+```
```

Comment:
since we have a demo plugin, can you change the exaples to use `console-demo-plugin` instead of `example-plugin` ? 
IMO its just easier to start if we show the whole process on demo plugin.

```suggestion
Run Bridge locally and instruct it to proxy e.g. `/api/plugins/console-demo-plugin` requests directly to your
local plugin asset server (web server hosting the plugin's generated assets):

```sh
./bin/bridge -plugins console-demo-plugin=http://localhost:9001/
```
```

Reply:
Agreed, will do.